### PR TITLE
Add AppDynamics browser RUM for React stack

### PR DESCRIPTION
**Pull request by redbugz**
We are going to try this out and see if it can give us valuable data or not.

### CODE REVIEW

File: packages/react-scripts/layout/views/layout.ejs [hunk 1]
```diff
@@ -19,6 +19,22 @@
     <!-- Resource Hints -->
     <link rel="preconnect" href="//edge.fscdn.org" crossorigin />
 
+    <% if (feature('appdynamicsFt') || process.env.APPDYNAMICS_ENABLED !== 'false') { %>
```
Comment (joeycozza) [inline packages/react-scripts/layout/views/layout.ejs]: I feel like the check of `APPDYNAMICS_ENABLED === 'true'` is a little more explicit in someone meant to turn it on. I know everyone probably gets the "false" from the defaults, so it probably doesn't really matter.

Comment (gmdayley) [reply packages/react-scripts/layout/views/layout.ejs]: I was going to make the same comment :)

Comment (redbugz) [reply packages/react-scripts/layout/views/layout.ejs]: I was trying to make the absence of the config work the same as true, but it doesn't really matter

Comment (redbugz) [reply packages/react-scripts/layout/views/layout.ejs]: typically for opt-out I would make an APPDYNAMICS_DISABLED=true check, but we want AppD opt-in for normal apps, but opt-out for React apps for now

Comment (redbugz) [reply packages/react-scripts/layout/views/layout.ejs]: in the future, the `||` will move to an `&&` as well.I just wanted an easy way to enable AppD RUM on the React apps to capture some data without all of them having to override the shared experiment.
So I expect this code will change over time.
Plus if we use it for real, we will have some functions that log custom data like user info, session info, etc.

**Pull request merged**